.github/workflows/pr-validation.yml

```diff
@@ -34,6 +34,30 @@ jobs:
       - name: Validate java code
         run: mvn clean verify -Dmaven.test.skip=true
 
+  run-format-and-lint-ui:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      
+      - name: Set up Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '16.x'
+
+      - name: Install dependencies
+        working-directory: ./ui
+        run: |
+          NODE_PATH=/cache/node_modules
+          NODE_OPTIONS=--max_old_space_size=6096
+          npm install --unsafe-perm --legacy-peer-deps
+
+      - name: Format and Lint all files
+        working-directory: ./ui
+        run: |
+          npm run format
+          npm run lint
+
   run-cypress-e2e-tests:
     runs-on: ubuntu-latest
     needs: [python-quality-checks, python-unit-tests]
```

Comment:
please add `run-format-and-lint-ui` here

Reply:
Done! :)